File: solved/X Campamento Caribeno/Contest 0/F.py
import sys

ans = [0 for i in range(222)]

ans[0] = 1
ans[2] = 1
ans[4] = 2
i = 6
while i < 222:

    for k in range(1, i):
        ans[i] += ans[k - 1] * ans[i - k - 1]

    i += 2

for line in sys.stdin:
    n = int(line)
    if (n > 0):
        print(ans[n * 2])
    else:
        break

File: solved/X Campamento Caribeno/Contest 4/F.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStream;
import java.io.InputStreamReader;
import java.math.BigInteger;
import java.util.StringTokenizer;


public class F {

	public static void main(String []arg) throws IOException  {
		Scanner sc = new Scanner(System.in);
		Task solver = new Task();
		solver.solve(sc);

	}
}
class Task{
	public void solve(Scanner in) {
		int a = in.nextInt();
		BigInteger it = new BigInteger(in.nextBiginteger(), a);
		int b = in.nextInt();
		String ans = it.toString(b);
		System.out.print(ans);
	}
}

class Scanner {
	public BufferedReader reader;
	public StringTokenizer st;

	public Scanner(InputStream stream) {
		reader = new BufferedReader(new InputStreamReader(stream));
		st = null;
	}

	public String next(){
		while(st == null || !st.hasMoreTokens()) {
			try{
				String line = reader.readLine();
				if(line == null) return null;
				st = new StringTokenizer(line);
			}catch (Exception e) {
				throw (new RuntimeException());
			}
		}
		return st.nextToken();
	}

	public int nextInt() {
		return Integer.parseInt(next());
	}

	public String nextBiginteger() {
		return next();
	}
}

